README.md

```diff
@@ -323,5 +326,8 @@ Letter templates(8.5in * 11in):
 * [29x29 - stores 24-word seeds ONLY as plaintext SeedQR format ONLY](docs/seed_qr/printable_templates/29x29_letter_trading_card_2sided.pdf)
 ---------------
 
-# Manual Installation Instructions
-see the docs: [Manual Installation Instructions](docs/manual_installation.md)
+# Developer Build Instructions
```

Comment:
Minor comment: maybe swap this and the "Build from Source" section since the SeedSigner OS route is the more important path that more people are likely to take.